lib/select.js

```diff
@@ -184,9 +185,9 @@ var classBase = React.createClass({
       this.onChange();
 
       if (!this.state.open) {
-        React.findDOMNode(this.refs['currentOption']).focus(); //eslint-disable-line dot-notation
+        this.refs['currentOption'].focus(); //eslint-disable-line dot-notation
       } else {
-          React.findDOMNode(this.refs['option' + (this.state.selectedOptionIndex || 0)]).focus();
+          ReactDOM.findDOMNode(this.refs['option' + (this.state.selectedOptionIndex || 0)]).focus();
```

Comment:
I tried removing this, but it threw a `this.state.selectedOptionIndex || 0)].focus(); is not a function` error.